projects/modular_llm/train_experts.py

```diff
@@ -203,7 +203,7 @@ def upload_library(expert_library, module):
                             expert = module.get_expert_instance(expert_name)
                             expert_library.add_expert(expert, expert_name)
                 elif isinstance(module, ExpertModule):
-                    expert = module.as_expert()
+                    expert = module.model.as_expert()
```

Comment:
What changed here for this to be required ?

Reply:
addressed